person.rb

```diff
@@ -1,29 +1,30 @@
 require './corrector'
+require './rental'
+
 class Person
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
-  private
-
-  def of_age?
-    @age >= 18
-  end
-
-  public
-
   def can_use_services?
     of_age? || @parent_permission
   end
 
   def validate_name
     @name = @corrector.correct_name(@name)
   end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
 end
```

Comment:
- [ ] <b>REQUIRED:</b> A method for adding rentals should be present` in person class. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 

```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```